Gui/opensim/view/src/org/opensim/view/motions/MotionDisplayer.java

```diff
@@ -376,7 +375,7 @@ public Vec3 getDefaultForceColorVec3() {
     
     /** Creates a new instance of MotionDisplayer */
     public MotionDisplayer(Storage motionData, Model model) {
-        String saved = "5";
+        String saved = "10";
```

Comment:
Please rename the variable to what is supposed to represent, e.g. `defaultMarkerRadius`